index.css

```diff
@@ -69,3 +69,216 @@
   font-size: 16px;
   line-height: 24px;
 }
+
+.achivement {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.recent {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+.works {
+  display: grid;
+}
+
+.storie h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.storie p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.tools {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  padding: 0;
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+.tools li {
+  list-style: none;
+  padding: 10px;
+  margin-right: 10px;
+  border: 1px solid black;
+}
```

Comment:
- [ ] According to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2154) the font family here should be `Inter`, kindly change that. 😁😁